packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -125,6 +125,11 @@ const MenuItem = styled.div<MenuItemProps>`
       : ''};
 `;
 
+const getUserSettingsMenuItemLinkStyles = () => css`
+  display: inline-block;
+  width: 100%;
```

Comment:
I think this is the same as:
```suggestion
  display: block;
```

Reply:
That is true. I think the tests should proove it: b46b4d4